code/modules/mob/living/simple_animal/hostile/ooze.dm

```diff
@@ -371,7 +371,7 @@
 	desc = "It somehow heals those who touch it."
 	icon = 'icons/obj/xenobiology/vatgrowing.dmi'
 	icon_state = "globule"
-	embedding = list("embed_chance" = 100, ignore_throwspeed_threshold = TRUE, "pain_mult" = 0, "jostle_pain_mult" = 0, "fall chance" = 0.5)
+	embedding = list("embed_chance" = 100, ignore_throwspeed_threshold = TRUE, "pain_mult" = 0, "jostle_pain_mult" = 0, "fall_chance" = 0.5)
```

Comment:
is there any reason for these to not be defines

Reply:
I am unsure about why the original author for the code did this, but other embed items seem to use code like this without defines either.